CONTRIBUTING.md

```diff
@@ -0,0 +1,107 @@
+# Contributing
+
+Contributions are **welcome** and will be fully **credited**.
+
+We accept contributions via Pull Requests on [Github](https://github.com/phpflo/phpflo).
+
+
+## Pull Requests
+
+- **[PSR-2 Coding Standard](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md)** - Check your code against the PSR-2 coding standard
+
+- **Add tests!** - Your patch won't be accepted if it doesn't have tests and don't follow our unit tests coding standards.
+
+- **Document any change in behaviour** - Make sure the `README.md` and any other relevant documentation are kept up-to-date.
+
+- **Consider our release cycle** - We try to follow [SemVer v2.0.0](http://semver.org/). Randomly breaking public APIs is not an option.
+
+- **Create feature branches** - Don't ask us to pull from your master branch.
+
+- **One pull request per feature** - If you want to do more than one thing, send multiple pull requests.
+
+- **Send coherent history** - Make sure each individual commit in your pull request is meaningful. If you had to make multiple intermediate commits while developing, please [squash them](http://www.git-scm.com/book/en/v2/Git-Tools-Rewriting-History#Changing-Multiple-Commit-Messages) before submitting.
+
+
+## Tests
+
+### Coding standards
```

Comment:
Some of theses guidelines cannot be tested statically by sniffer tools, a human check will always be required because the balance sometimes between solutions can be subjective. Even if it does automatically check errors, it seems mandatory to me to provide a lil bit of explanations behind the reasing of phpflo rules. Which is why it's important to be fully described in one place. And I don't think it makes sense to print the whole explanation in the sniffer tool output everytime it detects an error so we can't rely on that.